public/features.json

```diff
@@ -96,5 +112,20 @@
   },
   {
     "id": "stealth"
+  },
+  {
+    "id": "cleanse"
+  },
+  {
+    "id": "shield"
+  },
+  {
+    "id": "anti_shield"
+  },
+  {
+    "id": "anti_heal"
+  },
+  {
+    "id": "percentage_dmg"
```

Comment:
You should specify title for this and other names with underscores. Or you could add more complex logic to `Feature` entity replacing underscores with whitespaces